docs/en/Loading-Under-File-Storage-Metadata.md

```diff
@@ -0,0 +1,101 @@
+should behave the same as accessing them directly through the UFS. For example,
+if the root UFS is `s3://bucket/data`, listing `alluxio:///` should show the
+same result as listing `s3://bucket/data`, and running `cat` on `alluxio:///file`
+should show the same result as running `cat` on `s3://bucket/data/file`.
+
+To efficiently achieve this transparency, Alluxio lazily loads metadata from the
+UFS. In the above example, Alluxio will not know about `s3://bucket/data/file` at
+startup. `file` will only be discovered when the user lists `alluxio:///` or tries
+to cat `alluxio:///file`. This makes mounting a new UFS very cheap, and prevents
+unnecessary work.
+
+By default, *Alluxio expects that all modifications to UFSes happen through Alluxio*.
+This allows Alluxio to only scan each UFS directory a single time, significantly
+improving performance when UFS metadata operations are slow. As long as all UFS
+changes go through Alluxio, this has no user-facing impact.
+However, it is sometimes necessary for Alluxio to handle out of band UFS changes.
+That is where the metadata sync feature comes into play.
+
+## UFS Metadata Sync
+
+> The UFS metadata sync feature has been available since version `1.7.0`.
+
+When Alluxio scans a UFS directory and loads metadata for its sub-paths, it
+creates a cached copy of the metadata so that future operations don't need to
+hit the UFS. By default the cached copy lives forever, but you can configure the
+cache timeout with the `alluxio.user.file.metadata.sync.interval` configuration
+property. This is a client-side property, so when a client makes a request, it
+can specify `alluxio.user.file.metadata.sync.interval=1m` to say "when running
+this operation, make sure all relevant metadata has been synced within the last
+1 minute". Setting a value of `0` means that metadata will always be synced. The
+default value of `-1` means to never re-sync metadata.
+
+Low values for `alluxio.user.file.metadata.sync.interval` allow Alluxio clients
+to quickly observe out of band UFS changes, but increase the number of calls to
+the UFS.
+
+Metadata sync keeps a fingerprint of each UFS file so that Alluxio can re-sync
+the file if it changes. The fingerprint includes information such as file size
+and last modified time. If a file is modified in the UFS, Alluxio
+will detect this from the fingerprint, free the existing data for that file, and
+re-load the metadata for the updated file. If a file is deleted in the UFS, Alluxio
+will delete the file from Alluxio's namespace as well. Lastly, Alluxio will detect
+newly added files and make them available to Alluxio clients.
+
+## Techniques for managing UFS sync
+
+### Daily Metadata Sync
+
+If your UFS is only updated once a day, you can run
+
+`alluxio fs ls -R -Dalluxio.user.file.metadata.sync.interval=0 /`
+
+after the update, then use the default `-1` for the rest of the day to avoid calls
+to the UFS.
+
+### Centralized Configuration
```

Comment:
This section is for giving recommendations about how to use UFS sync to achieve common high-level goals. I think it's related to UFS sync since the two features work particularly well together, and users using UFS sync will often want to combine it with centralized configuration. If centralized configuration was more involved I would just link to it instead, but I think it's simple enough to inline here.